### ISSUE TITLE
Forum: Highlight most recently updated threads

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4112]
Related to #4093 entering the forum as CM i mostly want to see latest posts.

Click on `Forum Activities` gives me a list who posted when but not in which thread.

The presumably most common use case would be
- a list of most recently created / posted to threads (sort by descending date).
- with one last post directly below the title
- the list could be grouped by day or just show updates within 24 hours.

**Comment by dmtrjsg** [comment]
@traumschule 

Should we not  exactly the same treatment for latest posts as here: 
<img width="1308" alt="Screenshot 2023-01-27 at 12 36 20" src="https://user-images.githubusercontent.com/93924525/215088954-3d55388f-511e-473b-a744-9a0b5d011938.png">

We could also sort "My threads" by 
- Most posted
- Recently Updated
- Date Created

wdyt?



But with Recently updated only from My threads>

**Comment by dmtrjsg** [comment]
Action: We suggest to add sorting for My threads

- Most posted
- Recently Updated
- Date Created

**Comment by traumschule** [comment]
> Should we not exactly the same treatment for latest posts as here:

Yes! Implemented in #4881. Sorting would probably not result in the same, see https://github.com/Joystream/pioneer/issues/3029#issuecomment-3471267584
